### PR TITLE
Include GA combo tracking

### LINKED ISSUE #202: snapcraft.io should be added to the GA "combo tracker"
## Summary

The combo tracker in Google Analytics tracks all Ubuntu websites, snapcraft.io should be part of it.

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

Added GA tracking id

## QA

- Check code
- Test in GA if you have the creds

## Issue / Card

Fixes #202

### CODE REVIEW

**Comment by nottrobin** [comment]
From [Working with multiple trackers](https://developers.google.com/analytics/devguides/collection/analyticsjs/creating-trackers#working_with_multiple_trackers):

> To track data for two separate properties, you need to create two separate trackers, and **at least one of them must be a named tracker**. The following two commands create a default tracker and a tracker named "clientTracker":
>
> ``` javascript
> ga('create', 'UA-XXXXX-Y', 'auto');
> ga('create', 'UA-XXXXX-Z', 'auto', 'clientTracker');
> ```

If we're adding a second tracking code, I think one of them needs a name.

File: _includes/analytics.html [hunk 1]
```diff
@@ -19,6 +19,7 @@
   /**
    * Register our Google Analytics ID
    */
+  ga('create', 'UA-1018242-59', 'auto');
```
Comment (nottrobin) [inline _includes/analytics.html]: I think this will need a name.

Comment (pmahnke) [inline _includes/analytics.html]: I have added one Robin.

**Comment by anthonydillon** [comment]
@nottrobin can you check this resolves your issue.

**Comment by nottrobin** [comment]
@anthonydillon I approved my review already. It's fine. I'll merge.

**Pull request merged**